tests/RazorpayTest.php

```diff
@@ -27,6 +28,10 @@ public function testPayments()
         $this->assertTrue(is_array($data->toArray()));
 
         $this->assertTrue(is_array($data['items']));
+
+        $payment = $data['items'][0];
+
+        $this->assertTrue(get_class($payment['notes']) === Entity::class);
```

Comment:
Shouldn't we instead use StdClass for notes? Feels more logical rather than entity.